### PR TITLE
feat: react typescript syntax error message

### LINKED ISSUE #215: When you run after a React syntax error, give a good syntax error message in the errors panel
Image

Maybe babel can help us get a good syntax error, not sure.

/bounty $15

### PR DESCRIPTION
**Pull request by kom-senapati**
Closes #215 

image

I have added a util function and used it within the hook to return the react syntax error message.

/claim #215

### CODE REVIEW

File: src/hooks/use-run-tsx/index.tsx [hunk 1]
```diff
@@ -57,6 +58,17 @@ export const useRunTsx = ({
       })
     }
     if (!code) return
+    const error = getSyntaxError(code);
```
Comment (seveibar) [inline src/hooks/use-run-tsx/index.tsx:61]:
```suggestion
    const syntaxError = getSyntaxError(code);
```

File: src/hooks/use-run-tsx/index.tsx [hunk 2]
```diff
@@ -57,6 +58,17 @@ export const useRunTsx = ({
...
+    console.log("Syntax error: ", error)
```
Comment (seveibar) [inline src/hooks/use-run-tsx/index.tsx:62]:
```suggestion
```

**Review by seveibar — CHANGES_REQUESTED** [review]

File: src/hooks/use-run-tsx/index.tsx (hunk 1 above)
Comment (kom-senapati) [reply src/hooks/use-run-tsx/index.tsx:61]: Ok

File: src/hooks/use-run-tsx/index.tsx (hunk 2 above)
Comment (kom-senapati) [reply src/hooks/use-run-tsx/index.tsx:62]: Aah forgot it

**Comment by seveibar** [comment]
Can you make sure the tests pass and the format check passes, tyty

**Comment by kom-senapati** [comment]
I don't I did code changes for tests to break, I think its some existing issue
Could you please check @seveibar

**Review by seveibar — APPROVED** [review]

**Comment by kom-senapati** [comment]
Can you merge this @seveibar

**Pull request merged**